model/components/commercial/splunk.yml

```diff
@@ -0,0 +1,136 @@
+---
+components:
+
+    # Splunk
+  - id: splunk
+    name: Splunk
+    description: By combining Splunk Infrastructure Monitoring, APM, RUM, Log Observer, and On-Call, the Splunk Observability Suite offers a tightly integrated experience, powerful analytics, and unmatched speed and scale. Gain full-fidelity, end-to-end visibility of your apps, insights into root-causes of incidents and seamless troubleshooting for an elevated end-user experience.
+    link: https://splunk.com/
+    twitter: splunk
+    logo: splunk
+    categories:
+      - commercial-services
+    capabilities:
+      aspects:
+        - tracing
+        - metrics
+        - logs
+    connections:
+      includesFrom:
+        - jaeger-format
+        - zipkin-format
+      dataTo:
+        - grafana
+      dataFrom:
+        - opencensus
+        - opentelemetry
+        - jaeger
```

Comment:
Can you please change `jaeger` to either `jaeger-collector` and/ or `jaeger-agent`?